dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'HT': [Haiti, Haitian*, Porto Príncipe, Porto do Príncipe]
+    'JM': [Jamaica, Jamaican*, Kingston]
+    'KN': [São Cristóvão e Neves, São Cristóvão e Névis, São-Cristovense, Basseterre]
+    'KY': [Ilhas Cayman, Ilhas Caimã, Cayman, Caimão, Caimanês*, Caimanes*]
+    'LC': [Santa Lúcia, Santa-Lucense*, Santa-Luciense*, Castries]
+    'MF': [São Martinho, São-Martinhense*, Marigot]
+    'MQ': [Martinica, Martinican*]
+    'MS': [Monserrate, Monserratense*, Brades, Brades Estate]
+    'PR': [Porto Rico, Porto-Riquenh*, Porto-Riquense*, San Juan, São João]
+    'SX': [São Martinho, São-Martinhense*, Philipsburg]
+    'TC': [Ilhas Turcas e Caicos, Turquense*, Turco-Caicense*, Cockburn Town]
+    'TT': [Trindade e Tobago, Trinidade e Tobago, Trinidad e Tobago, Trinitári*, Trinitin*, Trinitário-Tobagense*, Tobaguian*, Port of Spain, Porto da Espanha, Porto de Espanha]
+    'VC': [São Vicente e Granadinas, São-Vicentin*, Kingstown]
+    'VG': [Ilhas Virgens Britânicas, Virginense*, Road Town]
+    'VI': [Ilhas Virgens Americanas, Ilhas Virgens dos Estados Unidos, Virginense*, Charlotte Amalie, Carlota Amália]
+
+  CENTER:
+    'BZ': [Belize, Belizenh*, Belizense*, Belmopan, Belmopã]
+    'CR': [Costa Rica, Costa-Riquenh*, Costa-Riquense*, Costarriquense*, Costa-Ricense*, Costarricense*, Ticos, San José, São José]
+    'GT': [Guatemala, Guatemaltec*, Guatemalense*, Guatemalês*, Guatemales*, Cidade da Guatemala]
+    'HN': [Honduras, Hondurenh*, Tegucigalpa]
+    'MX': [México, Estados Unidos Mexicanos, Mexican*, Cidade do México]
+    'NI': [Nicarágua, Nicaraguense*, Nica, Manágua]
+    'PA': [Panamá, Panamenh*, Panamense*, Cidade do Panamá]
+    'SV': [El Salvador, Salvadorenh*, San Salvador, São Salvador]
+
+  SOUTH:
+    'AR': [Argentina, Argentin*, Buenos Aires]
+    'BO': [Bolívia, Bolivian*, Sucre, La Paz]
+    'BR': [Brasil, Brasileir*, Brasília, São Paulo, Rio de Janeiro]
+    'CL': [Chile, Chilen*, Santiago, Santiago do Chile]
+    'CO': [Colômbia, Colombian*, Bogotá]
+    'EC': [Equador, Equatorian*, Quito]
+    'FK': [Ilhas Malvinas, Ilhas Falkland, Falklands, Falklander*, Malvinense*]
+    'GF': [Guiana Francesa, Franco-Guianense*]
+    'GY': [Guiana, Guyana, Guianês*, Guianense*]
+    'PE': [Peru, Peruan*, Peruvian*, Lima]
+    'PY': [Paraguai, Paraguai*, Paraguaian*, Assunção]
+    'SR': [Suriname, Surinamense*, Surinamês*, Paramaribo]
+    'UY': [Uruguai, Uruguai*, Montevidéu, Montevideu]
+    'VE': [Venezuela, Venezuelan*, Caracas]
+
+  NORTH:
+    'BM': [Bermudas, Bermudenh*, Bermudense*]
+    'CA': [Canadá, Canadense*, Canadian*, Ottawa, Otava, Toronto, Quebec, Cidade de Quebec]
+    'GL': [Gronelândia, Groenlândia, Groelândia, Gronelandês*, Groenlandês, Nuuk, Nuque]
+    'PM': [São Pedro e Miquelão, São Pedro e Miquelon, Saint-Pierre e Miquelon, Miquelonês*, São-Pedrense*, São Pedro]
+    'US': [Estados Unidos, Estados Unidos da América, América, EUA, Washington, Nova Iorque, Nova York]
```

Comment:
Should add demonyms? 

Maybe norte-americano*, estadunidense*, estado-unidense*, ianque*

Maybe remove América? Or it will create a lot of false positives for mentions of South America and North America.

Reply:
Changed